sdk/python/tests/integration/registration/test_universal_types.py

```diff
@@ -43,14 +43,19 @@ def populate_test_configs(offline: bool):
                     and feature_is_list is True
                 ):
                     continue
-                configs.append(
-                    TypeTestConfig(
-                        entity_type=entity_type,
-                        feature_dtype=feature_dtype,
-                        feature_is_list=feature_is_list,
-                        test_repo_config=test_repo_config,
+                for list_is_empty in [True, False]:
+                    configs.append(
+                        TypeTestConfig(
+                            entity_type=entity_type,
+                            feature_dtype=feature_dtype,
+                            feature_is_list=feature_is_list,
+                            list_is_empty=list_is_empty,
+                            test_repo_config=test_repo_config,
+                        )
                     )
-                )
+                    # For non list features `list_is_empty` does nothing
+                    if feature_is_list is False:
```

Comment:
No this is correct. I still want one just not the second one from the loop. Was a bit of an ugly hack in hindsight but was the fewest lines of code 😛

Reply:
haha can you instead do this before adding the config? It almost looks like a bug, and should be functionally equivalent.

Reply:
No, because then no config will get added 😕. For `feature_is_list is False` we still need to add one config it just doesn't matter is `list_is_empty` is `True` or `False`.

Reply:
Could make it 
```
if feature_is_list is False and list_is_empty is True:
    continue
```

Reply:
Done.